### PR TITLE
Enhance and separate RTL support

### PR DESCRIPTION
**Pull request by sohibegit**
Separated it as I want to override some of bootstrap CSS

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [X] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/files-angular.js [hunk 1]
```diff
@@ -58,6 +58,13 @@ const files = {
                 'content/css/_vendor.css',
                 'content/css/_documentation.css'
             ]
+        },
+        {
+            condition: generator => !generator.useSass && generator.enableI18nRTL,
```
Comment (gmarziou) [inline generators/client/files-angular.js]: Does it mean that enableI18nRTL is a global option for the whole app? 
If so, can we build an app with both a mix of LTR and RTL languages?

Comment (deepu105) [inline generators/client/files-angular.js]: yes, the styles are applied based on the language choosen at run time

Comment (gmarziou) [inline generators/client/files-angular.js]: Cool :)

**Comment by deepu105** [comment]
@sohibegit could you please also add a test case for this in the test-languages.js under test directory?

**Comment by sohibegit** [comment]
@deepu105 sure

**Pull request merged**